pkg/monitoring/perfscale/vmi-phase-transitions.go

```diff
@@ -128,12 +130,19 @@ func newVMIPhaseTransitionTimeHistogramVec(informer cache.SharedIndexInformer) *
 	return histogramVec
 }
 
-func updateVMIPhaseTransitionTimeFromCreationTimeHistogramVec(histogramVec *prometheus.HistogramVec, oldVMI *v1.VirtualMachineInstance, newVMI *v1.VirtualMachineInstance) {
+func updateVMIPhaseTransitionTimeFromCreationDeletionTimeHistogramVec(histogramVec *prometheus.HistogramVec, oldVMI *v1.VirtualMachineInstance, newVMI *v1.VirtualMachineInstance) {
```

Comment:
i think it would be best not to try to give this function multiple meanings.  The from creation and from deletion histograms should work independently of one another.